Description: Try to enable altivec on powerpc.
 Some compiler flags are not compatible with gcc, which need to fix.
 However, we didn't manage to build it, so we disable altivec in
 004_disable_altivec_on_ppc_linux.patch, while keep this patch for future use.
--- a/client/Makefile.am
+++ b/client/Makefile.am
@@ -165,7 +165,7 @@
 seti_boinc-analyzeFuncs_altivec.o:  vector/analyzeFuncs_altivec.cpp
 	if $(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) \
 	$(CPPFLAGS) $(seti_boinc_CXXFLAGS) $(CXXFLAGS) -DUSE_ALTIVEC \
-	-MT $@ -MD -MP -MF "$(DEPDIR)/$*.Tpo" -faltivec \
+	-MT $@ -MD -MP -MF "$(DEPDIR)/$*.Tpo" -maltivec \
 	-c -o $@ $<; then mv -f "$(DEPDIR)/$*.Tpo" "$(DEPDIR)/$*.Po"; \
 	else rm -f "$(DEPDIR)/$*.Tpo"; exit 1; fi
 seti_boinc-analyzeFuncs_vector.o:  vector/analyzeFuncs_vector.cpp
